src/kernel/kshell.c

```diff
@@ -317,6 +323,8 @@ void run_command(const char* command) {
         uptime();
     } else if (strncmp(command, "selftest", 8) == 0) {
         selftest();
+    } else if (strncmp(command, "overflow", 8) == 0) {
```

Comment:
We should probably document this command as well.

Reply:
Done